frontend/src/locales/en.json

```diff
@@ -99,6 +100,19 @@
     },
     "your_amount": "Your amount"
   },
+  "gdd_per_link": {
+    "copy": "copy",
+    "created": "Link Is Created!",
+    "decay-14-day": "Decay for 14 days",
+    "header": "Send Gradidos by link",
+    "link-copied": "Link copied to clipboard",
+    "not-copied": "Could not copy link: {err}",
+    "sentence_1": "Select an amount that you would like to send via a link. You can also enter a message. Click 'Generate now' to create a link that you can send.",
+    "sentence_2": "The link is valid for 14 days!",
+    "sentence_3": "If the link is not called up within the 14-day period, it will be deleted automatically. The amount will then be credited to your account.",
```

Comment:
```suggestion
    "sentence_3": "If the link is not redeemed within the 14-day validity period, it will be invalidated automatically. The amount will then be available to your account again.",
```